website/docs/getting-started/template-walkthrough.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 2
+---
+
+# Template Walkthrough
+
+The template generates several files, most notably:
+
+- `./package.json` includes `scripts` to `validate`, `synth`, and `deploy` your project.
+- `./tsconfig.json` configures the `@functionless/language-service` plugin to improve IDE feedback.
+- `./cdk.json` points CDK to the application entrypoint and requires the `functionless/register` hook to transpile project files.
+- `./src/app.ts` is the appliction entrypoint.
```

Comment:
Plan to add `jest.config.ts` in a follow-up change?

Reply:
Yup will be added in follow up.

Reply:
Issue?